### PR TITLE
telegram_user_id/telegram_bot_id fields

### PR DESCRIPTION
**Pull request by ice-myles**
telegram_user_id/telegram_bot_id fields and to be used at ModifyUser endpoint.

### CODE REVIEW

File: users/users_modify.go [hunk 1]
```diff
@@ -298,6 +300,16 @@ func (u *User) genSQLUpdate(ctx context.Context, agendaUserIDs []UserID) (sql st
 		sql += fmt.Sprintf(", agenda_contact_user_ids = $%v", nextIndex)
 		nextIndex++
 	}
+	if u.TelegramUserID != nil && *u.TelegramUserID != "" {
```
Comment (ice-ares) [inline users/users_modify.go]: U also need to allow removal of those fields

Comment (ice-myles) [reply users/users_modify.go]: Then they are need to have userID values by default, and I need to add new boolean ClearTelegramInfo field to set them to userID to reset them, right?

File: users/DDL.sql [hunk 2]
```diff
@@ -15,6 +15,7 @@ CREATE TABLE IF NOT EXISTS users  (
                     email text NOT NULL UNIQUE,
                     first_name text,
                     last_name text,
+                    telegram_bot_id text,
```
Comment (ice-ares) [inline users/DDL.sql:18]: Not null

File: users/DDL.sql [hunk 3]
```diff
@@ -47,10 +49,12 @@ END $$;
 ALTER TABLE users ADD COLUMN IF NOT EXISTS kyc_step_blocked smallint NOT NULL DEFAULT 0;
 ALTER TABLE users ADD COLUMN IF NOT EXISTS kyc_steps_last_updated_at timestamp[];
 ALTER TABLE users ADD COLUMN IF NOT EXISTS kyc_steps_created_at timestamp[];
-INSERT INTO users (created_at,updated_at,phone_number,phone_number_hash,email,id,username,profile_picture_name,referred_by,city,country,mining_blockchain_account_address,blockchain_account_address, lookup)
-                         VALUES (current_timestamp,current_timestamp,'bogus','bogus','bogus','bogus','bogus','bogus.jpg','bogus','bogus','RO','bogus','bogus',to_tsvector('bogus')),
-                                (current_timestamp,current_timestamp,'%[1]v','%[1]v','%[1]v','%[1]v','%[1]v','%[1]v.jpg','%[1]v','%[1]v','RO','%[1]v','%[1]v',to_tsvector('%[1]v')),
-                                (current_timestamp,current_timestamp,'icenetwork','icenetwork','icenetwork','icenetwork','icenetwork','icenetwork.jpg','icenetwork','icenetwork','RO','icenetwork','icenetwork',to_tsvector('icenetwork'))
+ALTER TABLE users ADD COLUMN IF NOT EXISTS telegram_user_id text;
```
Comment (ice-ares) [inline users/DDL.sql:52]: Text not null

File: users/DDL.sql [hunk 4]
```diff
@@ -47,10 +49,12 @@ END $$;
...
+ALTER TABLE users ADD COLUMN IF NOT EXISTS telegram_bot_id text;
```
Comment (ice-ares) [inline users/DDL.sql:53]: Text not null

**Pull request merged**